src/ui/public/utils/__tests__/parse_interval.js

```diff
@@ -64,8 +67,5 @@ describe('parseInterval', function () {
 
     duration = parseInterval('234asdf');
     expect(duration).to.not.be.ok();
-
-    duration = parseInterval('ms');
-    expect(duration).to.not.be.ok();
```

Comment:
Removing this test makes sense, but the `parseInterval('ms')` case should probably be added to the test block above (lines 47-59).  I assume it behaves the same as d, m, y, M.